_1327/settings.py

```diff
@@ -26,6 +26,9 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
+ENABLE_DEBUG_TOOLBAR = False
```

Comment:
please move to own commit and finish the implementation :)

Reply:
I picked these changes away for now :)